usecase/class/delete.go

```diff
@@ -0,0 +1,20 @@
+package class
+
+import (
+	"context"
+
+	"github.com/teq-quocbang/course-register/util/myerror"
+)
+
+func (u *UseCase) Delete(ctx context.Context, id string) error {
```

Comment:
Update lại logic, đầu tiên là getByID check xem id truyền có hợp lệ không => tìm ko thấy thì trả record not found => sau đó mới delete. Như source echo có example